apps/ui/src/components/pages/dashboard/getApplicationStateProps.ts

```diff
@@ -18,85 +18,85 @@
  */
 
 import { pcglColors } from '@/components/providers/ThemeProvider';
-import { ApplicationState } from '@/global/types';
+import { ApplicationStates, ApplicationStateValues } from '@pcgl-daco/data-model/src/types';
 
 type ApplicationCardProps = {
 	showEdit: boolean;
 	showActionRequired: boolean;
 	color: string;
 };
 
-export const getFriendlyStateName = (applicationState: ApplicationState): string => {
+export const getFriendlyStateName = (applicationState: ApplicationStateValues): string => {
 	switch (applicationState) {
-		case ApplicationState.DRAFT:
+		case ApplicationStates.DRAFT:
 			return 'Draft';
-		case ApplicationState.INSTITUTIONAL_REP_REVIEW:
+		case ApplicationStates.INSTITUTIONAL_REP_REVIEW:
 			return 'Rep Review';
-		case ApplicationState.DAC_REVIEW:
+		case ApplicationStates.DAC_REVIEW:
 			return 'DAC Review';
-		case ApplicationState.REP_REVISION:
+		case ApplicationStates.REP_REVISION:
 			return 'Rep Revisions';
-		case ApplicationState.DAC_REVISIONS_REQUESTED:
+		case ApplicationStates.DAC_REVISIONS_REQUESTED:
 			return 'DAC Revisions';
-		case ApplicationState.REJECTED:
+		case ApplicationStates.REJECTED:
 			return 'Rejected';
-		case ApplicationState.REVOKED:
+		case ApplicationStates.REVOKED:
 			return 'Revoked';
-		case ApplicationState.APPROVED:
+		case ApplicationStates.APPROVED:
 			return 'Approved';
-		case ApplicationState.CLOSED:
+		case ApplicationStates.CLOSED:
```

Comment:
@surajgoraya Now that I think about it, do we need this since "ApplicationStateValues" is from the pcgl package?

Reply:
I actually just got rid of this, since I currently was doing it by getting values from the translation file, i thought that'd be a better way. Take a look at apps/ui/src/i18n/locale/en/enTranslations.json and Line 79 in application card https://github.com/Pan-Canadian-Genome-Library/daco/pull/113/files#diff-758fec47d1b5d45e9b28529fb1777fc98bc5023d61db44f598ad9ecd2a935d7aR79

@JamesTLopez

Reply:
we can refactor to use StateValues if you think thats better?

Reply:
Ahh I see, I think removing it like you did was fine. We can just continue with ApplicationStateValues